### DISCUSSION TITLE
osu!Lazer beatmap listing search bar not working

### DISCUSSION THREAD

**Discussion by thekitsunedev** [ppy/osu#22162] [Q&A] [answered]
### Type

Game behaviour

### Bug description

I'm trying to download TheFatRat, Daft Punk, etc. maps and whenever I search for anything I get everything but not what I want

### Screenshots or videos

![osu_2023-01-12_19-53-37](https://user-images.githubusercontent.com/101592804/212168750-8ea1c1e5-7e98-4615-bdf2-3816bc6f64c8.jpg)
![osu_2023-01-12_19-52-59](https://user-images.githubusercontent.com/101592804/212168755-efd511a5-9042-4341-9779-c01130f4876e.jpg)

### Version

2022.1228.0-lazer

### Logs

.

**Comment by Susko3** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/22009.

**Reply by frenzibyte** [reply]
Note that changes have been made to make this more clarified (see https://github.com/ppy/osu/pull/22044), which will air next release.
